docs/resources/project.md

```diff
@@ -0,0 +1,45 @@
+# Artifactory Project Resource
+
+Provides an Artifactory project resource. This can be used to create and manage Artifactory project, maintain users/groups/roles/repos.
+
+## Example Usage
+
+```hcl
+# Create a new Artifactory project called my-project
+resource "project" "my-project" {
+  key                      = "myprj"
+  display_name             = "My Project"
+  max_storage_in_gibabytes = 10
+
+  admin_privileges {
+    manage_members   = true
+    manage_resources = true
+    index_resources  = true
+  }
+
+  member {
+    name  = "user1"
+    roles = ["developer","project admin"]
+  }
+}
+```
+
+## Argument Reference
+
+Project argument have an almost one to one mapping with the [JFrog Projects API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-PROJECTS). The following arguments are supported:
+
+* `key` - (Required)
```

Comment:
some details here? And, you know, there HAS to be a away to get the field descriptions out of terraform from the CLI - maybe figure that out so this can be auto generated?

Reply:
I may figure out the auto-generation in the next ticket